wren-ai-service/src/__main__.py

```diff
@@ -37,7 +37,23 @@ async def lifespan(app: FastAPI):
         engine_config=EngineConfig(provider=os.getenv("ENGINE", "wren_ui"))
     )
     container.init_globals(
-        *providers, should_force_deploy=bool(os.getenv("SHOULD_FORCE_DEPLOY", ""))
+        *providers,
+        should_force_deploy=bool(os.getenv("SHOULD_FORCE_DEPLOY", "")),
+        column_indexing_batch_size=(
+            int(os.getenv("COLUMN_INDEXING_BATCH_SIZE"))
+            if os.getenv("COLUMN_INDEXING_BATCH_SIZE")
+            else 50
+        ),
+        table_retrieval_size=(
+            int(os.getenv("TABLE_RETRIEVAL_SIZE"))
+            if os.getenv("TABLE_RETRIEVAL_SIZE")
+            else 10
+        ),
+        table_column_retrieval_size=(
+            int(os.getenv("TABLE_COLUMN_RETRIEVAL_SIZE"))
+            if os.getenv("TABLE_COLUMN_RETRIEVAL_SIZE")
+            else 1000
+        ),
```

Comment:
just mention first, and I don't think it need to be done in this PR. we can consider to use a config class to initial all env and it will make the code more concise.